src/index.ejs

```diff
@@ -4,6 +4,19 @@
 		<meta charset="UTF-8">
 		<title>L4GG | Timeline</title>
 		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta property="og:title" content="L4GG | Russiagate Timeline" />
```

Comment:
Sorry I missed this earlier--but I think L4GG wanted to get away from the title "Russiagate" @sgeofrank